index.bs

```diff
@@ -105,28 +104,33 @@ viewed along the positive direction of the axis (see figure below).
 Reference Frame {#reference-frame}
 ----------------
 
-The reference frame for {{Gyroscope}} is defined with
-a [=local coordinate system=], which can be defined as
-either the [=device coordinate system=], or the
-[=screen coordinate system=].
+The [=local coordinate system=] represents the reference frame for the
+[=Gyroscope=] [=sensor readings|readings=]. It can be either
+the [=device coordinate system=] or the [=screen coordinate system=].
 
 API {#api}
 ===
 
 The Gyroscope Interface {#gyroscope-interface}
---------------------------------
+------------------------
 
 <pre class="idl">
-  [Constructor(optional SpatialSensorOptions sensorOptions), SecureContext, Exposed=Window]
+  enum LocalCoordinateSystem { "device", "screen" };
+
+  dictionary GyroscopeSensorOptions : SensorOptions  {
```

Comment:
Should the dictionary definition go after interface definition, like in other specs?
@anssiko ?